src/components/Modals/Send/utils.ts

```diff
@@ -232,7 +234,11 @@ export const handleSend = async ({
         if (!adapter.signAndBroadcastTransaction) {
           throw new Error('signAndBroadcastTransaction undefined for wallet')
         }
-        return adapter.signAndBroadcastTransaction?.({ txToSign, wallet })
+        return adapter.signAndBroadcastTransaction({
+          senderAddress: from,
```

Comment:
`from` will be undefined for EVM Txs

<img width="1319" alt="image" src="https://github.com/shapeshift/web/assets/17035424/270244d2-2b0c-4e90-a43a-2b05efe7aefb">

```suggestion
        const { account } = fromAccountId(accountId)

        return adapter.signAndBroadcastTransaction({
          senderAddress: from ?? account,
```